### DISCUSSION TITLE
Requesting Enhancement to specify where minimized windows are docked

### DISCUSSION THREAD

**Discussion by dinaselim** [nextapps-de/winbox#141] [Ideas]
We use jqueryLayout to split the screen into left and right sides. The left side functions as a navigator and the right side is where we have the main functionality. When we minimize the window it is placed over both sides. Would be nice to provide the ability to specify where the minimized icons are placed.

**Comment by h3rb** [comment]
You can set the "containment element" to limit to one side for windows in your "main area"

**Comment by h3rb** [comment]
See some enhancements: https://github.com/h3rb/winbox

**Comment by ts-thomas** [comment]
Splitscreen is already supported. Look at the demo page. When using split screen the minimized windows just apply in their section.
